backend/app/services/knowledge/orchestrator.py

```
@@ -0,0 +1,900 @@
+                logger.warning(f"Task {task_id} has no teamRef")
+                return None
+
+            team_ref = task_crd.spec.teamRef
+            team = (
+                db.query(Kind)
+                .filter(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Team",
+                    Kind.name == team_ref.name,
+                    Kind.namespace == team_ref.namespace or "default",
+                    Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)